### PR TITLE
tslint curly, one variable per declaration, triple equals fixes for packages/base

### PR DESCRIPTION
**Pull request by cnishina**
- reference issue #1749

### CODE REVIEW

**Comment by cnishina** [comment]
- Made fixes to missing indentation.

Note to reviewer:

- Changes made from == to === should be verified that it is okay to compare with ===

File: packages/base/src/backbone-patch.ts [hunk 1]
```diff
@@ -44,7 +44,9 @@ import * as utils from './utils';
 // This *MUST* be called with the model as the `this` context.
 export
 function set(key, val, options) {
-    if (key == null) return this;
```
Comment (jasongrout) [inline packages/base/src/backbone-patch.ts]: This is a direct patch of the backbone source, so I'd rather keep the `==` here for better compatibility.

**Comment by jasongrout** [comment]
One inline comment - other than that, I think it looks fine. Thanks!

**Comment by jasongrout** [comment]
Thanks again! I fixed the one comment I made, and will merge when tests pass.

**Pull request merged**